src/client/components/componentWithSliders/componentWithSliders.css

```diff
@@ -25,3 +25,7 @@
   display: flex;
   justify-content: space-around;
 }
+
+.SlidersComponent .SavePresetButton {
+  margin-left: 0px;
```

Comment:
If smth is 0 in css, no need to specify which units you are using, if it’s px, rem or etc, it’s still be 0

Reply:
done)